ursgal/wrappers/msfragger_20170103.py

```diff
@@ -266,19 +269,19 @@ def preflight(self):
             self.params['input_file']
         )
         if self.input_file.lower().endswith('.mzml') or \
-                self.input_file.lower().endswith('.mzml.gz') or \
-                self.input_file.lower().endswith('.mgf'):
+                self.input_file.lower().endswith('.mzml.gz'):# or \
+                # self.input_file.lower().endswith('.mgf'):
             self.params['translations']['mzml_input_file'] = self.input_file
-        # elif self.input_file.lower().endswith('.mgf'):
-        #     self.params['translations']['mzml_input_file'] = \
-        #         self.meta_unodes['ucontroller'].get_mzml_that_corresponds_to_mgf( self.input_file )
-        #     self.print_info(
-        #         'MSFragger can only read Proteowizard MGF input files,'
-        #         'the corresponding mzML file {0} will be used instead.'.format(
-        #             os.path.abspath(self.params['translations']['mzml_input_file'])
-        #         ),
-        #         caller = "INFO"
-            # )
+        elif self.input_file.lower().endswith('.mgf'):
+            self.params['translations']['mzml_input_file'] = \
+                self.meta_unodes['ucontroller'].get_mzml_that_corresponds_to_mgf( self.input_file )
+            self.print_info(
+                'MSFragger can only read Proteowizard MGF input files,'
+                'the corresponding mzML file {0} will be used instead.'.format(
+                    os.path.abspath(self.params['translations']['mzml_input_file'])
+                ),
+                caller = "INFO"
+            )

```

Comment:
Why not allow mgf files, if we explicitly pass them to the wrapper?

Reply:
Because it doesn't work for any mgf files that I have.
If you have mgf files that work, we can keep it in, or try to figure out how they need to look like. But otherwise it seems like a function that just doesn't work.

Reply:
I thought you had figured this out at some point, wasn't there something with 6 decimal points in the mgf or similar?

Reply:
I thought I used mgfs with MSfragger after you found a fix, but I could also be wrong ...

Reply:
what about this d8d6083691dd85fcdb1790fc1d765dbb613c9ce3 commit?

Reply:
Yeah, it had worked for a while but then not anymore and I'm not sure if it's an msfragger version issue or an issue with the mgf/mzML file conversion.

Reply:
So, it seems like the earlier versions of MSFragger can't deal with "our" mgfs, but the new version (I've tested 2.3 so far) seems to work. So I would still keep the change here to use mzMLs but then re-introduce mgf usage with the 2.3 wrapper.